docs/source/backends/vpn/openvpn.rst

```diff
@@ -2,7 +2,7 @@
 OpenVPN 2.3 Backend
 ===================
 
-.. include:: ../_github.rst
+.. include:: /_github.rst
```

Comment:
Hey Gagan, I think we should not do this because these pages are indexed by search engines and if we move them around without providing redirects we'll lose traffic. I think it's not worth doing.

Reply:
Do you mean restructuring VPN docs or just this include part?

Reply:
restructuring, that is, rename/move the files, we should avoid it